packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -222,6 +222,19 @@ const start = async (zcf, privateArgs) => {
     zcf.reallocate(offerToSeat, collateralSeat);
   };
 
+  const burnRUNToReduceShortfall = reduction => {
+    reduction = AmountMath.coerce(runBrand, reduction);
+    const runKeyword = keywordForBrand.get(runBrand);
+    const runBalance = collateralSeat.getAmountAllocated(runKeyword);
+    const amountToBurn = AmountMath.min(reduction, runBalance);
+    if (AmountMath.isEmpty(amountToBurn)) {
+      return;
+    }
+
+    runMint.burnLosses(harden({ [runKeyword]: amountToBurn }), collateralSeat);
```

Comment:
does this still allow that the assertReserve contract's keywords are private and meaningless?

Reply:
it should.  Just above this, `runKeyword` is set using `const runKeyword = keywordForBrand.get(runBrand);`, so whatever we use for the keywords should still be in `keywordForbrand`.

Reply:
cool. I wasn't sure how keywords interacted with burnLosses but I see they're ignored https://github.com/Agoric/agoric-sdk/blob/fd44fca1ae25f9b17c34404bc03a7e79bdb4982b/packages/zoe/src/contractFacet/zcfZygote.js#L174-L175